module/segment/src/Infrastructure/Persistence/Query/DbalSegmentProductsQuery.php

```diff
@@ -42,6 +44,37 @@ public function getProducts(SegmentId $segmentId): array
             ->fetchAll(\PDO::FETCH_COLUMN);
     }
 
+    /**
+     * @return array
+     */
+    public function getAllEditedProducts(SegmentId $segmentId, ?\DateTime $dateTime = null): array
+    {
+        $qb = $this->getQuery();
+
+        $qb->select('sp.product_id')
+            ->join('sp', self::AUDIT_TABLE, 'a', 'a.id = sp.product_id')
+            ->where($qb->expr()->eq('segment_id', ':segmentId'))
+            ->andWhere($qb->expr()->eq('available', ':available'))
+            ->setParameter(':available', true)
+            ->setParameter(':segmentId', $segmentId->getValue());
+
+        if ($dateTime) {
+            $qb
+                ->andWhere($qb->expr()->gte('edited_at', ':editedAt'))
+                ->setParameter(':editedAt', $dateTime, Types::DATETIMETZ_MUTABLE);
+        }
+
+        $result = $qb
+            ->execute()
+            ->fetchAll(\PDO::FETCH_COLUMN);
```

Comment:
```suggestion
            ->fetchAllAssociative();
```

Reply:
This method is used alongside $this->productQuery->getAllEditedIds in ProductExportProcessStep and there is used 
            ->fetchAll(\PDO::FETCH_COLUMN);
Data formats in results       ->fetchAll(\PDO::FETCH_COLUMN); and  ->fetchAllAssociative(); is different. 

Should I create task for changing this in both places?